sys-apps/likwid/likwid-4.1.2.ebuild

```diff
@@ -0,0 +1,101 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+FORTRAN_NEEDED=fortran
+
+inherit fcaps linux-info fortran-2
+
+DESCRIPTION="A performance-oriented tool suite for x86 multicore environments"
+HOMEPAGE="https://github.com/rrze-likwid/likwid"
+SRC_URI="https://ftp.fau.de/pub/likwid/${P}.tar.gz"
+
+LICENSE="GPL-3+"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="fortran"
+
+RDEPEND="dev-lang/perl"
+
+DEPEND="${RDEPEND}
+	fortran? ( sys-devel/gcc:*[fortran] )
+	dev-lang/lua:0"
+
+CONFIG_CHECK="~X86_MSR"
+
+FILECAPS=(
+	-M 755 cap_sys_rawio usr/sbin/likwid-accessD --
+	-M 755 cap_sys_rawio usr/bin/likwid-{perfctr,bench,powermeter}
+)
+
+# See Bug 558402
+PATCHES=(
+	"${FILESDIR}/${P}-Makefile.patch"
+	"${FILESDIR}/${P}-fix-gnustack.patch"
+	"${FILESDIR}/${P}-config.mk.patch"
```

Comment:
I am confused.
You asked me not to include the patches in the commit before. 
Also ${P} is correct. Why hard code the version number.
I never used ${PN} for the patches in this ebuild.

Example patch name "likwid-4.1.2-config.mk.patch"  (for new patches)
For last version -
https://gitweb.gentoo.org/repo/gentoo.git/tree/sys-apps/likwid/files